src/WebUI/Common/KubeSummary.scss

```diff
@@ -1,15 +1,33 @@
 .kubernetes-container {
     // cards have margins
-    .k8s-card-padding+.k8s-card-padding {
+    .k8s-card-padding + .k8s-card-padding {
         margin-top: 16px;
     }
-
     /*Css to adjust the alignment of text between picklist and text search. Revisit when we move from picklist to dropdown (fabric-free)*/
-    .keyword-search.bolt-text-field{
+    .keyword-search.bolt-text-field {
         padding-bottom: 8px;
     }
 
     .loading-pods {
         height: 200px;
     }
-}
\ No newline at end of file
+
+    .details-card-row-size {
+        padding-top: 6px;
+        padding-bottom: 6px;
+        display: flex;
+    }
+
+    .details-card-info-field-size {
+        margin-right: 8px;
+        width: 150px;
+        min-width: 150px;
+    }
+
+    .details-card-value-field-size {
+        margin-right: 8px;
+        position: absolute;
+        width: calc(100% - 158px);
+        margin-left: 158px;
+    }
```

Comment:
make margin-right as 20px, 
width: calc(100% - 116px)
margin-left is 116px

Reply:
Done. Have made changes to ensure the UI paddings/margins/fonts are same as in figma